README.md

```diff
@@ -98,6 +98,30 @@ what could be perceived as a loss of resolution by resorting to aggregation on t
 in the Datadog Agent already aggregates messages over a certain flush period, therefore so long as the flush interval configured configured on the client side is smaller
 than said flush interval on the server side there should no loss in resolution.
 
+### Configuration 
+
+Enabling aggregation is simple, you just need to set the appropriate options with the client builder.
+
+You can just enable aggregation by calling the `enableAggregation(bool)` method on the builder. 
+
+There are two clent-side aggregation knobs available:
+- `aggregationShards(int)`: determines the number of shards in the aggregator, this
+ feature is aimed at mitigating the effects of map locking in highly concurrent scenarios.
+- `aggregationFlushInterval(int)`: sets the period of time in milliseconds in which the
+aggregator will flush its metrics into the sender. Defaults to 3000 milliseconds.
+
+```java
+
+StatsDClient client = new NonBlockingStatsDClientBuilder()
+    .prefix("statsd")
```

Comment:
unrelated nit: looks like the `prefix` call is also mentioned in the main `Configuration` part of the readme, probably for parity with the "deprecated" constructor mentioned, but I don't think the `prefix` option should be mentioned in basic instantiation examples, especially with a value with low meaning like `statsd` (would make sense in advanced docs though).